### PR TITLE
Remove " />" from admin

### PR DESCRIPTION
**Pull request by dbltoe**
These are the affected files in the admin EXCEPT the includes/fonts and includes/javascript/flot files.

### CODE REVIEW

**Comment by scottcwilson** [comment]
Great!  Thank you.

File: admin/includes/modules/dashboard_widgets/RecentOrdersDashboardWidget.php [hunk 1]
```diff
@@ -72,7 +72,7 @@
                 <?php echo $order['orders_id'] . ' - ' . substr($order['customers_name'], 0, 20); ?>
             </a>
           </td>
-          <td class="text-right" title="<?php echo zen_output_string($product_details, array('"' => '&quot;', "'" => '&#39;', '<br />' => '', '<hr>' => "----\n")); ?>">
+          <td class="text-right" title="<?php echo zen_output_string($product_details, array('"' => '&quot;', "'" => '&#39;', '<br>' => '', '<hr>' => "----\n")); ?>">
```
Comment (torvista) [inline admin/includes/modules/dashboard_widgets/RecentOrdersDashboardWidget.php:75]: I think br=>'' should be in addition to the br /, not replace it.

File: admin/includes/modules/dashboard_widgets/RecentOrdersDashboardWidget.php [hunk 2]
```diff
@@ -81,7 +81,7 @@
                  data-trigger="focus"
                  data-placement="left"
                  title="<?php echo TEXT_PRODUCT_POPUP_TITLE; ?>"
-                 data-content="<?php echo zen_output_string($product_details, array('"' => '&quot;', "'" => '&#39;', '<br />' => '<br>')); ?>"
+                 data-content="<?php echo zen_output_string($product_details, array('"' => '&quot;', "'" => '&#39;', '<br>' => '<br>')); ?>"
```
Comment (torvista) [inline admin/includes/modules/dashboard_widgets/RecentOrdersDashboardWidget.php:84]: As pre previous comment.

File: admin/orders.php [hunk 3]
```diff
@@ -1341,7 +1341,7 @@ function couponpopupWindow(url) {
 <?php echo $orders->fields['delivery_state'] . '<br>' . $orders->fields['delivery_country']; ?>
                 </td>
 <?php } ?> 
-                <td class="dataTableContent text-right" title="<?php echo zen_output_string($product_details, ['"' => '&quot;', "'" => '&#39;', '<br />' => '', '<hr>' => "----\n"]); ?>">
+                <td class="dataTableContent text-right" title="<?php echo zen_output_string($product_details, ['"' => '&quot;', "'" => '&#39;', '<br>' => '', '<hr>' => "----\n"]); ?>">
```
Comment (torvista) [inline admin/orders.php:1344]: As per previous comment.

File: admin/orders.php [hunk 4]
```diff
@@ -1350,7 +1350,7 @@ function couponpopupWindow(url) {
                        data-trigger="focus"
                        data-placement="left"
                        title="<?php echo TEXT_PRODUCT_POPUP_TITLE; ?>"
-                       data-content="<?php echo zen_output_string($product_details, ['"' => '&quot;', "'" => '&#39;', '<br />' => '<br>']); ?>"
+                       data-content="<?php echo zen_output_string($product_details, ['"' => '&quot;', "'" => '&#39;', '<br>' => '<br>']); ?>"
```
Comment (torvista) [inline admin/orders.php:1353]: As per previous comment.

**Pull request merged**